Documentation/client.md

```diff
@@ -106,31 +106,31 @@ the s3cmd tool is included in the [Rook toolbox](toolbox.md) pod.
 1. Get the connection information for accessing object storage
 
    ```bash
-   eval $(rookctl object connection mystore rook-user --format env-var)
+   eval $(rookctl object connection my-store rook-user --format env-var)
    ```
 
 1. Create a bucket in the object store
 
    ```bash
-   s3cmd mb --no-ssl --host=${AWS_HOST} --host-bucket=  s3://rookbucket
+   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
```

Comment:
is this change from `AWS_HOST` to `AWS_ENDPOINT` strictly needed?  i.e. is this documentation stale and wrong, was there a code change in 0.6 that needs this change to the docs?

Reply:
i don't believe there was an 0.6 change related to this. AWS_HOST just wasn't working for me in the toolbox, but AWS_ENDPOINT did work.

Reply:
I thought this was part of the fix in https://github.com/rook/rook/pull/1172

Reply:
good reminder to check. The AWS_HOST is not working for me in the toolbox, so i'll investigate a bit more

Reply:
i confirmed that AWS_HOST does work ok, so I'll revert that change

Reply:
i can't repro the issue now. Both AWS_HOST and AWS_ENDPOINT are working in the toolbox for me